public/app/features/alerting/unified/components/rules/Filter/RulesFilter.v1.tsx

```diff
@@ -40,11 +42,6 @@ const ViewOptions: SelectableValue[] = [
     label: 'List',
     value: 'list',
   },
-  {
-    icon: 'heart-rate',
-    label: 'State',
-    value: 'state',
-  },
```

Comment:
(I didn't realise we were removing the state view in this PR btw, are we all okay for it to be removed? As I know there were recent customer tickets relating to it, so we know people are definitely using it)

Reply:
No, we shouldn't. These new view type buttons should be hidden behind the feature flag